### PR TITLE
Alert, button components added

### PR DESCRIPTION
**Pull request by muskan-gupta-0803**

### CODE REVIEW

File: components/Alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,36 @@
+@import "/styles.css";
+@import "/docs.css";
+
+.alert-box{
+    width:90%;
+    padding:1rem;
+    margin:1rem;
+    border-radius:5px;
+    font-size: 1rem;
+
```
Comment (dev-enforced) [inline components/Alert/alert.css]: Remove the extra space

File: components/Alert/alert.html [hunk 2]
```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Alert</title>
+    <link rel="stylesheet" href="alert.css">
+    <link rel="shortcut icon" href="images/favicon-32x32.png" type="image/x-icon">
+    <!-- <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous"/> -->
```
Comment (dev-enforced) [inline components/Alert/alert.html]: Remove the commented code and extra space

File: components/Alert/alert.html [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+            </a>
+    
+            <!-- nav links -->
+            <div class="nav-links">
+                <a href="docs.html" class="nav-items"><span>Docs</span></a>
+                <a href="https://github.com/muskan-gupta-0803/LilacUI" class="nav-items"><span>Github</span></a>
+                <a href="#" class="nav-items"><span>
+                    <i class="fas fa-sun"></i>
+                </span></a>
+                <!-- <a href="#" class="nav-items"><span>Item 4</span></a> -->
```
Comment (dev-enforced) [inline components/Alert/alert.html]: Remove the commented code

File: components/Alert/alert.html [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+                <h5 class="mp-heading">On this page</h5>
+                <li class="mp-item"><a class="mp-link" href="#std-alert">Standard Alert</a> </li>
+                <li class="mp-item"><a class="mp-link" href="#icon-alert">Alert with Icon</a> </li>
+                
+            </ul>
+
+        </aside>
+    </div>
+
+    <footer class="footer">
```
Comment (dev-enforced) [inline components/Alert/alert.html:114]: If it isn't needed then you can remove it !!

File: components/Button/button.html [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+            </a>
+    
+            <!-- nav links -->
+            <div class="nav-links">
+                <a href="docs.html" class="nav-items"><span>Docs</span></a>
+                <a href="https://github.com/muskan-gupta-0803/LilacUI" class="nav-items"><span>Github</span></a>
+                <a href="#" class="nav-items"><span>
+                    <i class="fas fa-sun"></i>
+                </span></a>
+                <!-- <a href="#" class="nav-items"><span>Item 4</span></a> -->
```
Comment (dev-enforced) [inline components/Button/button.html]: Remove the commented code as above

File: components/Alert/alert.html [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+        </nav>
+    </header>
+
+    <div class="doc-container">
+    <!-- every page same menu -->
+        <aside class="menu-all">
+            <ul class="doc-list ma-list">
+                <h5 class="ma-heading">Get Started</h5>
+                <li class="ma-item"><a class="ma-link" href="docs.html">Introduction </a></li>
+                <li class="ma-item"><a class="ma-link" href="components/Utilities/utilities.html">Utilities </a></li>
```
Comment (dev-enforced) [inline components/Alert/alert.html:39]: Here when I click on any of the buttons then you can specify a path href="../utilities/utilities.html" and pass it to the list item

**Review by dev-enforced — CHANGES_REQUESTED** [review]
The code looks good to me !! Suggested some minor changes which should be corrected one by one
https://deploy-preview-2--lilac-ui.netlify.app/components/alert/alert
https://deploy-preview-2--lilac-ui.netlify.app/components/button/button
Great work on the components Muskan

File: components/Alert/alert.html (hunk 6 above)
Comment (muskan-gupta-0803) [reply components/Alert/alert.html:39]: okay i will change it later on.

**Review by dev-enforced — APPROVED** [review]
The changes look good !!Now you can merge it @muskan-gupta-0803 !! Good work on the components

**Review by pratik-kumar-sahu — APPROVED** [review]
- You can use colors as variables.
- Looks good to merge.

**Comment by muskan-gupta-0803** [comment]
> * You can use colors as variables.
> * Looks good to merge.

sure, thanks for the review

**Pull request merged**